apps/vdjbase-singularity/0.1/cedar/update-bundle.sh

```diff
@@ -0,0 +1,15 @@
+#
+TOOL=changeo-singularity
+SYSTEM=ls5
```

Comment:
cedar?

Reply:
Same as above, not used, but didn't want to delete.